src/hist/basehist.py

```diff
@@ -87,6 +98,42 @@ def Reg(self, *args, **kwargs):
         self._ax.append(Regular(*args, **kwargs))
         return self
 
+    @always_normal_method
+    def Sqrt(self, *args, **kwargs):
+        if self._hist:
+            raise RuntimeError("Cannot add an axis to an existing histogram")
+        self._ax.append(Regular(*args, transform=hist.axis.transform.sqrt, **kwargs))
+        return self
+
+    @always_normal_method
+    def Log(self, *args, **kwargs):
```

Comment:
For all these functions we are wrapping, we should look up and use the real Python 3 signature from boost-histogram's decorators. That way, tab completion and docs are better.